src/nemo_run/cli/api.py

```diff
@@ -928,7 +928,17 @@ def _should_continue(self, skip_confirmation: bool) -> bool:
         Returns:
             bool: True if execution should continue, False otherwise.
         """
-        return skip_confirmation or typer.confirm("Continue?")
+        env_skip = os.environ.get("NEMORUN_SKIP_CONFIRMATION", "").lower() == "true"
+
+        if skip_confirmation or env_skip:
+            os.environ["NEMORUN_SKIP_CONFIRMATION"] = "true"
+            return True
+
+        if typer.confirm("Continue?"):
+            os.environ["NEMORUN_SKIP_CONFIRMATION"] = "true"
+            return True
+
+        return False
```

Comment:
Will this work in a multiprocess setting. for eg. when launching with `torchrun`? I think each process has its own `os.environ` so this won't help the other processes. Can you test this in a multi process setting?